ocaml/xapi-aux/kerberos_encryption_types.ml

```diff
@@ -28,6 +28,21 @@ module Winbind = struct
     | All ->
         "all"
 
+  let to_int = function
+    (*
+     * [X] 0x00000001 DES-CBC-CRC
```

Comment:
The official docs for these bits is here I think: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-kile/6cfc7b50-11ed-4b4d-846d-6f08f0812919?redirectedfrom=MSDN

There are 5 more bits there that would be all 0 in our settings. They are not encryption related directly (except for J which forces an upgrade from RC4?), but would be good to understand whether we need them or not:

F | FAST-supported
G | Compound-identity-supported
H | Claims-supported
I | Resource-SID-compression-disabled
J | AES256-CTS-HMAC-SHA1-96-SK

Reply:
In fact the latest Samba manual lists 2 more:
https://www.samba.org/samba/docs/current/man-html/net.8.html
0x00000020 AES256-CTS-HMAC-SHA1-96-SK
0x00080000 RESOURCE-SID-COMPRESSION-DISABLED

Samba's default used to be 31, and is now 28 (although not sure what the default for our version of Samba is)

Reply:
our version of samba does not support them. (maybe XS9 does support, need verification)